.github/ISSUE_TEMPLATE/archived.md

```diff
@@ -3,19 +3,18 @@ name: Archiving checklist
 about: archiving
 title: "[ARCHIVE] project"
 labels: archive
-assignees: caniszczyk, amye, idvoretskyi, jeefy
+assignees: caniszczyk, mrbobbytables, idvoretskyi, jeefy, castrojo
 ---
 - [ ] TOC vote to archive passed
-- [ ] Move logo to archived on CNCF.io	
-- [ ] Remove from landscape		
-- [ ] Archive maintainers mailing list	
+- [ ] Move logo to archived on CNCF.io
+- [ ] Remove from landscape	
+- [ ] Archive maintainers mailing list
 - [ ] Remove ML from maintainers@ mailing group
 - [ ] Remove/Archive on maintainers.cncf.io
-- [ ] Remove logo from all slide decks	
-- [ ] Remove logo from event sites	
-- [ ] Move logo to archived in artwork	
-- [ ] Archive DevStats (or archive?)	
-- [ ] Notify Speakers team to remove from CFPs	
-- [ ] Remove from service desk or archive	
+- [ ] Remove logo from all slide decks
+- [ ] Remove logo from event sites
+- [ ] Archive DevStats
```

Comment:
Should add Insights here too, and potential other LFX tools

Reply:
yeah, I noticed another thing. I'll flag this as WIP for now